builders/interact/substrate-api/chopsticks.md

```diff
@@ -166,28 +164,68 @@ Chopstick's internal WebSocket server has special endpoints that allow the manip
 
 These are the methods that can be invoked and their parameters:
 
-|      Method      |      Parameters       |                          Description                          |
-|:----------------:|:---------------------:|:-------------------------------------------------------------:|
-|  `dev_newBlock`  |       `options`       |               Generates one or more new blocks.               |
-| `dev_setStorage` | `values`, `blockHash` |         Create or overwrite the value of any storage.         |
-| `dev_timeTravel` |        `date`         |     Sets the timestamp of the block to the `date` value.      |
-|  `dev_setHead`   |    `hashOrNumber`     | Sets the head of the blockchain to a specific hash or number. |
-
-??? code "Parameters details"
-
-    |   Parameter       |               Format                |                                Example                                 |
-    |:-----------------:|:-----------------------------------:|:----------------------------------------------------------------------:|
-    |   `options`       | `{ "to": number, "count": number }` |                            `{ "count": 5 }`                            |
-    |    `values`       |              `Object`               | `{ "Sudo": { "Key": "0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b" } }`  |
-    |  `blockHash`      |              `string`               | `"0x1a34506b33e918a0106b100db027425a83681e2332fe311ee99d6156d2a91697"` |
-    |     `date`        |               `Date`                |                        `"2030-08-15T00:00:00"`                         |
-    | `hashOrNumber`    |         `number | string`           |                                                                        |
-
-    - **`options` { "to": number, "count": number }** - a JSON object where `"to"` will create blocks up to a certain value, and `"count"` will increase by a certain number of blocks. Use only one entry at a time within the JSON object  
-    - **`values` Object** - a JSON object resembling the path to a storage value, similar to what you would retrieve via Polkadot.js  
-    - **`blockHash` string** - optional, the block hash at which the storage value is changed  
-    - **`date` Date** - a Date string (compatible with the JavaScript Date library) that will change the timestamp at which the next blocks being created will be. All future blocks will be sequentially created after that point in time  
-    - **`hashOrNumber` number | string** - if found, the chain head will be set to the block with the block number or block hash of this value  
+??? function "**dev_newBlock** (options) — Generates one or more new blocks"
```

Comment:
```suggestion
???+ function "**dev_newBlock** (options) — Generates one or more new blocks"
```

Typically, I leave the first one as expanded so it lets readers know they can expand them in case they may have any uncertainty